pkg/cmd/roachtest/tests/multitenant.go

```diff
@@ -82,60 +80,41 @@ func runAcceptanceMultitenant(ctx context.Context, t test.Test, c cluster.Cluste
 		r := strings.Split(regionAndZone, "-")
 		return r[0] + "-" + r[1]
 	}
 
-	const tenantID = 123
-	{
-		// Intentionally, alter settings so that the system database span config
-		// changes propagate faster, when we convert the system database to MR.
-		conn := c.Conn(ctx, t.L(), 1)
-		defer conn.Close()
-		_, err := conn.Exec(`SELECT crdb_internal.create_tenant($1::INT)`, tenantID)
+	// Start a virtual cluster.
+	const virtualClusterName = "multiregion-tenant"
+	virtualClusterNode := c.Node(1)
+	c.StartServiceForVirtualCluster(
+		ctx, t.L(),
+		option.StartVirtualClusterOpts(virtualClusterName, virtualClusterNode),
+		install.MakeClusterSettings(),
+	)
+
+	virtualClusterURLs := func() []string {
+		urls, err := c.ExternalPGUrl(ctx, t.L(), virtualClusterNode, roachprod.PGURLOptions{
+			VirtualClusterName: virtualClusterName,
+		})
 		require.NoError(t, err)
-		configStmts := []string{
-			`SET CLUSTER SETTING sql.virtual_cluster.feature_access.multiregion.enabled='true'`,
-			`SET CLUSTER SETTING kv.closed_timestamp.target_duration = '200ms'`,
-			`SET CLUSTER SETTING kv.rangefeed.closed_timestamp_refresh_interval = '200ms'`,
-			"SET CLUSTER SETTING kv.allocator.load_based_rebalancing = off",
-			"SET CLUSTER SETTING kv.allocator.min_lease_transfer_interval = '10ms'",
-			"SET CLUSTER SETTING kv.closed_timestamp.side_transport_interval = '50 ms'",
-			`SET CLUSTER SETTING kv.closed_timestamp.lead_for_global_reads_override = '1500ms'`,
-			`ALTER TENANT ALL SET CLUSTER SETTING spanconfig.reconciliation_job.checkpoint_interval = '500ms'`,
-			`SET CLUSTER setting kv.replication_reports.interval = '5s';`,
-		}
-		for _, stmt := range configStmts {
-			_, err := conn.Exec(stmt)
-			require.NoError(t, err)
-		}
-	}
 
-	const (
-		tenantHTTPPort  = 8081
-		tenantSQLPort   = 30258
-		otherRegionNode = 7
-	)
-	// Start an equal number of tenants on the cluster, located in the same regions.
-	tenants := make([]*tenantNode, 0, len(c.All()))
-	for i, node := range c.All() {
-		region := regions[i]
-		regionInfo := fmt.Sprintf("cloud=%s,region=%s,zone=%s", c.Cloud(), regionOnly(region), region)
-		tenant := deprecatedCreateTenantNode(ctx, t, c, c.All(), tenantID, node, tenantHTTPPort, tenantSQLPort, createTenantRegion(regionInfo))
-		tenant.start(ctx, t, c, "./cockroach")
-		tenants = append(tenants, tenant)
+		return urls
+	}()
 
-		// Setup the system database for multi-region, and add all the region
+	const otherRegionNode = 7
+	for i := range c.All() {
+		// Set up the system database for multi-region, and add all the region
 		// in our cluster.
 		if i == 0 {
 			includedRegions := make(map[string]struct{})
-			verifySQL(t, tenants[0].pgURL,
+			verifySQL(t, virtualClusterURLs[0],
 				mkStmt("SET CLUSTER SETTING sql.region_liveness.enabled='yes'"),
 			)
-			verifySQL(t, tenants[0].pgURL,
+			verifySQL(t, virtualClusterURLs[0],
 				mkStmt(fmt.Sprintf(`ALTER DATABASE system SET PRIMARY REGION '%s'`, regionOnly(regions[0]))),
```

Comment:
This doesn't work because the tenant user doesn't have access to modify the `system` db 🤔 

This might not be an issue with your test set up? It should be creating the default user and granting it `ADMIN` permissions, and in fact I see the comment in the logs saying: `log into DB console for virtual cluster multiregion-tenant with user=roachprod password=cockroachdb`.

Would need to take a deeper look to see why it's not working. Maybe the `pgurl` isn't correct? 

Anyway, if it is unrelated you can pass in `Auth: install.AuthRootCert` to `c.ExternalPGUrl()` as a temporary unblocker, although in the end we should get this working with non root.